torchci/test/drci.test.ts

```diff
@@ -29,8 +29,8 @@ export const successfulA = {
 };
 
 const pendingA = {
-  name: "linux-docs / build-docs (cpp)",
-  conclusion: null,
+  name: "linux-docs / build-docs (python)",
```

Comment:
Why was this change needed?

Reply:
Oh, this has the same name as the previous successful mock example `successfulA`.  So if I accidentally include both in my mock `successfulA` and `pendingA`.  Only the pending one is counted.  It's clearer to use another name here I guess.